Assignment1.1.1.c

```diff
@@ -0,0 +1,14 @@
+//Bettina Benitez
+#include <stdio.h>
```

Comment:
your code didn't handle floats like the directions said to! You also didn't include comments.